ext/snowcrash

```diff
@@ -1 +1 @@
-Subproject commit 65e01df77c0e7ad86efb3c0efa4cbc69c846db98
+Subproject commit 76560473da89b5ba4be0a4826288ced2a9712533
```

Comment:
I would prefer doing this after the release (just to make sure there are no other commits in the SC master).